packages/dx-react-grid-material-ui/src/templates/table-group-cell/icon.jsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import ChevronRight from '@material-ui/icons/ChevronRight';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import IconButton from '@material-ui/core/IconButton';
+import classNames from 'classnames';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  groupButton: {
+    verticalAlign: 'middle',
+    display: 'inline-block',
+    padding: theme.spacing.unit,
+    marginRight: theme.spacing.unit,
+  },
+});
+
+const IconBase = ({
+  expanded,
+  classes,
+  className,
+  ...restProps
+}) => (
+  <IconButton
+    className={classNames(classes.groupButton, className)}
+    {...restProps}
+  >
+    {
+      expanded
+        ? <ExpandMore />
+        : <ChevronRight />
+    }
+  </IconButton>
+);
+
+IconBase.propTypes = {
+  expanded: PropTypes.bool.isRequired,
+  className: PropTypes.string,
```

Comment:
We often add a property `classes` to `propTypes`. `classes: PropTypes.object.isRequired,`